### ISSUE TITLE
delete

### ISSUE THREAD

**Issue by henryi720349** [nvaccess/nvda#11400]
Pls delete

**Comment by Adriani90** [comment]
I cannot reproduce this. When I switch to focus mode with nvda+space bar and press tab until I land on "enlish", then I press down arrow, and then I press tab again, NVDA says "language list expanded". In Firefox it even says "expanded" and "colapsed" when I press down arrow or escape on the language selector. In Chrome it does not report this.
Pressing tab on the language list navigates me from language to language.
But I cannot choose a language by pressing enter, for that I have to use object navigation, route the mouse to the language and press the left mouse click.

I think NVDA does not fully support this kind of selector yet.

cc: @feerrenrut, @JulienCochuyt, I saw this kind of selectors many times in corporate environments where costomized browser based apps are used.

@ObjectInSpace, is the state "expanded" or "colapsed" exposed properly in Chrome? I wonder because in Firefox NVDA reports the state change but not in Chrome.

**Comment by JulienCochuyt** [comment]
I notice two issues with this example:
 - NVDA does not detect that the *drop-down list* is a candidate for focus mode: pressing the `enter` key does not trigger focus mode but `NVDA+space` naturally does.
 - The *drop-down list* when collapsed, appears as two controls, the second of which is not spoken at all: See #6627 and #11378.

I did not inspect the DOM of the page, but I'd bet this *drop-down list* is oddly designed.

**Comment by Adriani90** [comment]
Closing since missing description.
